lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2612,6 +2628,31 @@ def _add_to_dict(metadata):
         return d
 
 
+def loggly_time_string(dt):
+    r = dt.strftime("%Y-%m-%d") + "T" + dt.strftime("%H") + "%3A" + dt.strftime("%M") + "%3A"
+    r += str(dt.second) + str(round(dt.microsecond * (10.0**-5), 3)) + "Z"
+    return r
+
+
+def get_loggly_query_string(lbry_id):
+    decoded_id = base58.b58encode(lbry_id)
+    now = datetime.now()
+    yesterday = now.replace(day=now.day - 1)
```

Comment:
I don't think this works across month boundaries.  You want:
yesterday = `now - datetime.timedelta(days=1)`